### PR TITLE
[chore] Update node to fix docker image vulnerabilities

### PR DESCRIPTION
**Pull request by w1kman**
This PR aims to fix vulnerabilities inherited by `node:16-slim` image.

image

- Node `v16 => v18`
- CLI/Docker
  - Added `-s, --stdout` so that docker image can be used without mounting a volume
  - entrypoint defaults to be using `--stdout`
- Docs
  - Updated `README.md` to include docker usage

**How to test**
- Build docker image and run it as shown in `README.md`
  - Build: `docker build --tag har-to-k6:local .`
  - Run: `docker run har-to-k6:local ./example/inconsistent-content-type.har > loadtest.js`
    - Expected result is a `loadtest.js` file on the host system
  - Run: `docker run har-to-k6:local -o loadtest.js ./example/inconsistent-content-type.har`
    - Expected result is a `loadtest.js` file inside the container (old default).
  - Note: warnings and errors should not end up in the test script
- Run `./bin/har-to-k6` with `-stdout` and `--output` in different combinations to make sure that we get a desired result

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,4 +7,4 @@ COPY . .
 RUN npm install
 RUN npm run bundle
 
-ENTRYPOINT ["node", "bin/har-to-k6.js"]
+ENTRYPOINT ["node", "bin/har-to-k6.js", "--stdout"]
```
Comment (w1kman) [inline Dockerfile:10]: If the user would add option `-o, --output <filename>`, output would not be sent to `stdout`

Comment (w1kman) [inline Dockerfile:10]: Docker defaults to `stdout`. If `-o, --output` is used, `--stdout` is overridden.

File: webpack.config.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 /** @type {import('webpack').Configuration } */
 module.exports = {
   entry: './src/index.js',
-  node: { fs: 'empty' },
+  resolve: {
+    fallback: {
+      fs: false,
+    },
+  },
```
Comment (w1kman) [inline webpack.config.js:11]: Change required when updating `webpack`

**Review by w1kman — COMMENTED** [review]
Self-review

**Review by w1kman — COMMENTED** [review]
self-review#2

File: package.json [hunk 3]
```diff
@@ -56,13 +56,13 @@
...
-        "webpack-cli": "^3.3.4"
+        "webpack": "^5.88.2",
+        "webpack-bundle-analyzer": "^4.9.1",
+        "webpack-cli": "^5.1.4"
     },
     "scripts": {
-        "bundle": "./node_modules/.bin/webpack --config webpack.config.js",
-        "analyze": "BUNDLE_ANALYSIS=\"reports/commit_$(git rev-parse --short HEAD).html\" ./node_modules/.bin/webpack --config webpack.config.js",
+        "bundle": "NODE_OPTIONS=--openssl-legacy-provider ./node_modules/.bin/webpack --config webpack.config.js",
+        "analyze": "NODE_OPTIONS=--openssl-legacy-provider BUNDLE_ANALYSIS=\"reports/commit_$(git rev-parse --short HEAD).html\" ./node_modules/.bin/webpack --config webpack.config.js",
```
[the remark this answers is not in the document]
Comment (w1kman) [reply package.json]: Works fine in node `v16|v18` without `--openssl-legacy-provider`

File: package.json [hunk 4]
```diff
@@ -56,9 +56,9 @@
         "npm-run-all": "^4.1.5",
         "pretty-quick": "^2.0.1",
         "sinon": "^7.3.1",
-        "webpack": "^4.34.0",
-        "webpack-bundle-analyzer": "^3.8.0",
-        "webpack-cli": "^3.3.4"
+        "webpack": "^5.88.2",
+        "webpack-bundle-analyzer": "^4.9.1",
+        "webpack-cli": "^5.1.4"
```
Comment (allansson) [inline package.json:61]: Why you no change to parceljs? 😞

**Review by e-fisher — APPROVED** [review]
LGTM

**Pull request merged**